ramls/instance.json

```diff
@@ -460,6 +486,22 @@
             "type": "string",
             "description": "The human readable ID, also called eye readable ID. A system-assigned sequential ID which maps to the Instance ID"
           },
+          "isbnId": {	
+            "type": "string",	
+            "description": "isbn Id"	
+          },	
+          "issnId": {	
+            "type": "string",	
+            "description": "issn Id"	
+          },	
+          "isbnValue": {	
+            "type": "string",	
+            "description": "isbn value"	
+          },	
+          "issnValue": {	
+            "type": "string",	
+            "description": "issn value"	
+          },
```

Comment:
Why have properties been added to the preceding and succeeding titles as part of this change?

Reply:
This has happened because they are now part of the instance schema as expressed by [data-import-processing-core](https://github.com/folio-org/data-import-processing-core/blob/master/ramls/instance.json). After some investigation i have learned that at some point in the past the instance.json schema was copied out of mod-inventory into  data-import-processing-core, and then it was used to generate pojos. Since that time two things have happened:

1) Both the mod-inventory instance.json and the data-import-processing-core instance.json have each diverged.
2) mod-inventory has imported the generated classes from folio.org.

In an attempt to bring the canonical definition of the instance back into mod-inventory I have modified the pom to begin generating pojos from the local schema, and brought the instance.json changes required by data-import-processing-core back into mod-inventory.

Reply:
> In an attempt to bring the canonical definition of the instance back into mod-inventory I have modified the pom to begin generating pojos from the local schema, and brought the instance.json changes required by data-import-processing-core back into mod-inventory.

That is an admirable intention.

I don't think the public API of a module should be altered due to an internal need if at all possible.

Do you know why these fields have been added to the schema in data import processing core? If not, please can you follow up with the Folijet folks.

Can this be resolved by stopping generating code within mod-inventory and instead using hand rolled classes instead?

Reply:
Yes, so I spent today rethinking my approach, specifically because of the potential for conflict with the data processing stuff. I think I will push up new code on Monday that will avoid the generation of code and adhere more closely to the established implementation for InstanceRelationships. Hopefully, this will address these concerns.